Responsive desktop portfolio branch

ClintonYounge opened the pull request:
1. Added and linked a new CSS file responsible for the responsiveness needed for the desktop version of my portfolio. Utilizing media query.

2. Made the Opacity for the navigation buttons 100 in the media query. So they'll show after the screen in stretched.

ClintonYounge committed:
Update html and css

Added css file for responsive CSS. necessary for the desktop version

Added custom background

ClintonYounge committed:
Update html nav

ClintonYounge committed:
Update html and css

Added figma background image.
Fixex to html

ClintonYounge committed:
Update style.css

ClintonYounge committed:
Update buttons styling

ClintonYounge committed:
Fixed linter errors

pinkmoon25 commented on style.css:
```diff
@@ -3,10 +3,7 @@
+  box-sizing: border-box;
 }
 
 nav {
```
- [ ] Good job creating the desktop version and making it responsive, I think we can improve the navbar for our design to match the original template design, as one of the goal for this project is to parse a figma template as close as possible our current navbar is currently missing some padding and background color see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213489678-17675992-5f2b-4028-9ada-4325501a2d2f.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213490032-129d53fb-4237-4984-b869-6bbb5f67d8a1.png)

you see we can take the height like 70 as looking in the picture and adjust our padding top and bottom to be as close to it as possible also notice the background color is `#fff`

pinkmoon25 commented on responsive-desktop.css:
```diff
@@ -0,0 +1,246 @@
+    border-radius: 0;
+    margin: 50px 100px;
+    height: fit-content;
+  }
```
- [ ] As mentioned above please notice the original design for our projects section and how there is a border-radius property and also the cards width are wider, as this is the main section of our portfolio, it is important to show our project cards well, It's a good thing that you have used width as percentage, it will make it easier to make it wider and apply the `border-radius: 24px;` property as well 
see figma :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213492156-ae3ba9e9-da27-44a4-9bfb-dbc15a0bf043.png)

pinkmoon25 commented on responsive-desktop.css:
```diff
@@ -0,0 +1,246 @@
+    width: 480px;
+  }
+
+  #username,
+  #user-email,
+  #text-area {
+    width: 100%;
+    margin: 12px 0;
+    border: 1px solid #cfd8dc;
+    padding: 1rem 12px;
+  }
```
- [ ] Please notice in the figma template the border-raius for these inputs is 2px, currently you have it same as in the mobile version which is 10px, how about we fix this as well  :+1:

pinkmoon25 commented on responsive-desktop.css:
```diff
@@ -0,0 +1,246 @@
+  .language-list li {
+    flex-direction: column;
+    height: fit-content;
+  }
```
- [ ] I see we can improve our tech cards a bit to match the Figma template much more closely, our current design looks like :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213493958-39c3a82a-684f-46ff-a974-df7f7170a24f.png)

```
align-items: baseline;
padding: 16px;
```
adding the above properties to our tech lists it will make it like :point_down: 

![image](https://user-images.githubusercontent.com/97521557/213494464-307afbfa-cf3a-43be-b3a5-ab8948272909.png)

pinkmoon25 commented on index.html:
```diff
@@ -16,17 +17,20 @@
 
   <section class="introduction">
-    <h1 class="title"> I'm Clinton, Glad to See You!</h1>
+    <h2 class="title"> I'm Clinton, Glad to See You!</h2>
```
- [ ] [OPTIONAL] what do you think about adding a `<br>` tag here so that the design follows how it is in the figma template `<h2 class="title"> I'm Clinton,<br>Glad to See You!</h2>` will make it like :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213496051-ebd9d545-b26f-4745-a2c9-351d760ed5c6.png)

pinkmoon25 reviewed (changes requested):
Hi @ClintonYounge , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/Ns2K2ajFe2TT0Fx9bM/giphy.gif"/>

## Highlights
linter checks :heavy_check_mark: 
follows figma template :heavy_check_mark: 
Used flex and grid :heavy_check_mark: 
media query :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ClintonYounge commented:
Great review. Thanks a lot

On Thu, Jan 19, 2023, 12:25 PM Tushar Singh ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @ClintonYounge <https://github.com/ClintonYounge> ,
>
> Good job so far!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
>
> <https://camo.githubusercontent.com/d8da4dd5ccd8733549e516fe56162c301917f1164f5e7be0492160e0c26cca47/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f4e73324b32616a466532545430467839624d2f67697068792e676966>
> Highlights
>
> linter checks ✔️
> follows figma template ✔️
> Used flex and grid ✔️
> media query ✔️
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In style.css
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/4#discussion_r1081476320>
> :
>
> > @@ -3,10 +3,7 @@
>
>  *::after {
>
>    margin: 0;
>
>    padding: 0;
>
> -}
>
> -
>
> -body {
>
> -  background: #e5e5e53b;
>
> +  box-sizing: border-box;
>
>  }
>
>
>
>  nav {
>
>
>
>    - Good job creating the desktop version and making it responsive, I
>    think we can improve the navbar for our design to match the original
>    template design, as one of the goal for this project is to parse a figma
>    template as close as possible our current navbar is currently missing some
>    padding and background color see 👇
>    [image: image]
>    <https://user-images.githubusercontent.com/97521557/213489678-17675992-5f2b-4028-9ada-4325501a2d2f.png>
>
> how it should be 👇
> [image: image]
> <https://user-images.githubusercontent.com/97521557/213490032-129d53fb-4237-4984-b869-6bbb5f67d8a1.png>
>
> you see we can take the height like 70 as looking in the picture and
> adjust our padding top and bottom to be as close to it as possible also
> notice the background color is #fff
> ------------------------------
>
> In responsive-desktop.css
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/4#discussion_r1081486786>
> :
>
> > +  .projects {
>
> +    display: flex;
>
> +    flex-direction: row;
>
> +    padding: 24px;
>
> +    width: 70%;
>
> +    background: #fff;
>
> +    border: 1px solid #dfe1e6;
>
> +    border-radius: 0;
>
> +    margin: 50px 100px;
>
> +    height: fit-content;
>
> +  }
>
>
>
>    - As mentioned above please notice the original design for our
>    projects section and how there is a border-radius property and also the
>    cards width are wider, as this is the main section of our portfolio, it is
>    important to show our project cards well, It's a good thing that you have
>    used width as percentage, it will make it easier to make it wider and apply
>    the border-radius: 24px; property as well
>    see figma 👇
>    [image: image]
>    <https://user-images.githubusercontent.com/97521557/213492156-ae3ba9e9-da27-44a4-9bfb-dbc15a0bf043.png>
>
> ------------------------------
>
> In responsive-desktop.css
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/4#discussion_r1081506023>
> :
>
> > +  #username,
>
> +  #user-email,
>
> +  #text-area {
>
> +    width: 100%;
>
> +    margin: 12px 0;
>
> +    border: 1px solid #cfd8dc;
>
> +    padding: 1rem 12px;
>
> +  }
>
>
>
>    - Please notice in the figma template the border-raius for these
>    inputs is 2px, currently you have it same as in the mobile version which is
>    10px, how about we fix this as well 👍
>
> ------------------------------
>
> In responsive-desktop.css
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/4#discussion_r1081506185>
> :
>
> > +  .language-list li {
>
> +    flex-direction: column;
>
> +    height: fit-content;
>
> +  }
>
>
>
>    - I see we can improve our tech cards a bit to match the Figma
>    template much more closely, our current design looks like 👇
>    [image: image]
>    <https://user-images.githubusercontent.com/97521557/213493958-39c3a82a-684f-46ff-a974-df7f7170a24f.png>
>
> align-items: baseline;
>
> padding: 16px;
>
>
> adding the above properties to our tech lists it will make it like 👇
>
> [image: image]
> <https://user-images.githubusercontent.com/97521557/213494464-307afbfa-cf3a-43be-b3a5-ab8948272909.png>
> ------------------------------
>
> In index.html
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/4#discussion_r1081512091>
> :
>
> >      </nav>
>
>    </header>
>
>
>
>    <section class="introduction">
>
> -    <h1 class="title"> I'm Clinton, Glad to See You!</h1>
>
> +    <h2 class="title"> I'm Clinton, Glad to See You!</h2>
>
>
>
>    - [OPTIONAL] what do you think about adding a <br> tag here so that
>    the design follows how it is in the figma template <h2 class="title">
>    I'm Clinton,<br>Glad to See You!</h2> will make it like 👇
>    [image: image]
>    <https://user-images.githubusercontent.com/97521557/213496051-ebd9d545-b26f-4745-a2c9-351d760ed5c6.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/4#pullrequestreview-1262086277>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4JSSCCBI72KNCOKTQ3PJELWTFTGVANCNFSM6AAAAAAUAPDNDA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ClintonYounge committed:
Update html and css

1. Added a br in my html title.

2. Changed navbar color to match figma template.

3. displayed projects width increase to 90% and border radius increased to 24px.

4. edited form input boxes radius to 2px. To match figma template.

5. Changed nav height to 70px to match figma template.

ridwanediallo reviewed (approved):
# STATUS: APPROVED 🟢
## Hi @ClintonYounge , 
![Congratulations!GIF](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions
 
- [OPTIONAL ]  I think removing the background color you have added to your chevron icons would make them look nicer. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
